app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -117,7 +117,7 @@ def search_form_action
 
     def render_visibility_link(document)
       link_to render_visibility_label(document),
-              main_app.edit_curation_concerns_file_set_path(document, anchor: "permissions_display"),
+              main_app.edit_curation_concerns_generic_work_path(document, anchor: "permissions_display"),
```

Comment:
This is concerning to me.  Are we sure that document always represents a GenericWork and never a FileSet?  I'm pretty sure the FileSet page calls this method too.